openapi/golem-service.yaml

```diff
@@ -4515,13 +4515,24 @@ components:
       enum:
       - directory
       - file
+    GatewayBindingComponent:
+      type: object
+      properties:
+        name:
+          type: string
+        version:
+          type: integer
+          format: uint64
+      required:
+      - name
+      - version
     GatewayBindingData:
       type: object
       properties:
         bindingType:
           $ref: '#/components/schemas/GatewayBindingType'
-        componentId:
-          $ref: '#/components/schemas/VersionedComponentId'
+        component:
```

Comment:
Probably we need to keep things backward compatible. We can't break it at 1.2, because 1.1 users - almost all - during different hackathons used API definitions.

Reply:
We are totally breaking backward compatibility with the other changes we decided to do, so it does not matter